zerver/tests/test_subs.py

```diff
@@ -816,6 +816,34 @@ def test_successful_subscriptions_add(self):
                 add_streams, self.streams, self.test_email, self.streams + add_streams)
         self.assert_length(events, 4, True)
 
+    def test_successful_subscriptions_add_with_announce(self):
```

Comment:
Can you change the commit message to be more explicit that your commit adds test coverage?  I would probably just say `Add test_successful_subscriptions_add_with_announce()` or `Test that you can add streams multiple times.`